### DISCUSSION TITLE
"Stretch" auto changes to "Fit" so image is cropped

### DISCUSSION THREAD

**Discussion by ddrfraser1** [t1m0thyj/WinDynamicDesktop#468] [Q&A]
the bottom and top of my image is cut off unless I manually select "Strech" in Windows setings. But it automatically goes back to "Fit". Is there a way to fix?

**Comment by t1m0thyj** [comment]
If you are using the Microsoft Store version, this troubleshooting tip may help: [Wallpaper fit not saved with multiple monitors](https://github.com/t1m0thyj/WinDynamicDesktop/wiki/Known-issues#wallpaper-fit-not-saved-with-multiple-monitors)

**Reply by ddrfraser1** [reply]
That seems to have worked! Thanks!
